dev_tool/src/test/uploads.ts

```diff
@@ -0,0 +1,27 @@
+import { spawn } from 'child_process'
+import { requireBinary } from '../deps.js'
+
+function checkUploadsDeps() {
+    const helpString =
+        'ClamAV must be installed and in $PATH in order to test uploads. You can install with `brew install clamav`'
```

Comment:
Should we add `clamav` into the list of packages to install with `brew` under the `Local Tooling` section of our Readme?

Reply:
done